src/test/java/uk/nhs/prm/e2etests/performance/PerformanceTest.java

```diff
@@ -121,43 +136,61 @@ public void testAllSuspensionMessagesAreProcessedWhenLoadedWithProfileOfRatesAnd
             generateThroughputPlot(recorder, THROUGHPUT_BUCKET_SECONDS, "Suspension event mean throughput per second in " + THROUGHPUT_BUCKET_SECONDS + " second buckets");
         }
 
-        assertThat(recorder.hasUnfinishedEvents()).isFalse();
+        assertFalse(recorder.hasUnfinishedEvents());
     }
 
     private NemsTestEvent injectSingleNemsSuspension(NemsTestEventListener listener, NemsTestEvent testEvent) {
-        var nemsSuspension = testEvent.createMessage();
+        final String liversedgeMedicalCentreOdsCode = "B85612";
+        final String previousGP = randomOdsCode();
+        final String timestamp = LocalDateTime.now().toString() + "Z";  // ZonedDateTime.now(ZoneOffset.ofHours(0)).toString();
```

Comment:
Can we remove this comment?

```suggestion
        final String timestamp = LocalDateTime.now().toString() + "Z";
```